application/src/main/java/org/togetherjava/tjbot/config/FeatureBlacklist.java

```diff
@@ -0,0 +1,34 @@
+package org.togetherjava.tjbot.config;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+
+import java.util.Set;
+
+/**
+ * Blacklist of features, use {@link FeatureBlacklist#isEnabled(T)} to test if a feature is enabled.
```

Comment:
javadoc should briefly explain what a blacklisted feature is.

Reply:
Done, tell me what you think.